Documentation/UWP.md

```diff
@@ -0,0 +1,41 @@
+# Game class constructor
+Due to some UWP implementation details, Monogame has to construct your `Game` derived class by itself, using a static initializer `MonoGame.Framework.XamlGame<T>.Create(...)`.
+
+In this situation, we have to main possibilities to create a `Game` derived class:
+
+1. Let `XamlGame` to initialize your `Game` derived class using the default (parameterless!) constructor
+2. Let `XamlGame` to initialize your `Game` derived class using a custom (rich!) constructor.
+
+##### 1. XamlGame uses the default parameterless constructor
+
+With this logic, it isn't possible to inject in a "clear" way some dependencies, since the default constructor is called: `var game = new T();`
```

Comment:
I would write:
it isn't possible to inject **dependencies through the constructor**, since ...